docs/multisite/languages/back_office_translations.md

```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.
```

Comment:
If we want to mention the Bazinga  I would go with the full name `BazingaJsTranslationBundle`

Reply:
```suggestion
    Run `composer run post-update-cmd` which both install your translations to JavaScript using the `BazingaJsTranslationBundle`, and clear the default SiteAccess cache.
```